src/modules/package.rs

```diff
@@ -68,6 +68,10 @@ fn extract_package_version(file_contents: &str, display_private: bool) -> Option
     };
 
     let formatted_version = format_version(raw_version);
+    if formatted_version == "v0.0.0-development" || formatted_version.find("semantic").is_some() {
```

Comment:
Let's make it bit more strict because we might encounter actual prerelase version (non-zero) containing `semantic` identifier:
```suggestion
    if formatted_version == "v0.0.0-development" || formatted_version.starts_with("v0.0.0-semantic") {
```

Reply:
The current implementation is aiming to identify any of the following common semantic release placeholders:
- `0.0.0-semantically-released`
- `0.0.0-semantic-versioning`
- `0.0.0-semantic-release`

Do you know of any packages that currently have the `semantic` identifier in their version, and aren't using semantic-release?

Reply:
> The current implementation is aiming to identify any of the following common semantic-release placeholders:

That's perfectly fine I just wanted to make it more strict to prevent something like `1.4.21-foo-semantic-bar` being incorrectly matched. And yes, I'm not aware of anyone out there using such ridiculous prerelease identifiers but let's not underestimate people's creativity :stuck_out_tongue_winking_eye:

Reply:
Was thinking about this when I wrote this originally actually

My thinking was if they had `semantic` in a version like this then it's arguably still more likely (albeit speculative) that  they were using semantic versioning than not with something like `1.4.21-foo-semantic-bar` - though not sure how we'd be able to explain the `1.4.21`

Open to tweak it or keep the same, what do you think @matchai ?

Reply:
I think the safer option is to prefix with v0.0.0, and keep an eye out for issues opening up with exceptional edge cases 😊